tool/tsh/common/app_test.go

```diff
@@ -288,6 +288,16 @@ func TestAppCommands(t *testing.T) {
 									}, setHomePath(loginPath), webauthnLoginOpt)
 								}()
 
+								if requireMFAType == types.RequireMFAType_SESSION {
+									// mfa verified proxy certs should not be saved to disk.
+									err = Run(context.Background(), []string{
+										"app",
+										"config",
+										app.name,
+									}, setHomePath(loginPath))
+									require.Error(t, err)
```

Comment:
Should we put some additional expectations on this error? Like its type or message.